net/mwan3/files/etc/init.d/mwan3

```diff
@@ -17,6 +17,7 @@ start_tracker() {
 	interface=$1
 	config_get_bool enabled $interface 'enabled' '0'
 	[ $enabled -eq 0 ] && return
+        [ -z "$(config_get $interface track_ip)" ] && return
```

Comment:
tab vs space?

Reply:
thank you.

Reply:
@aaronjg I haven't tested it yet, so I'm not sure, but don't we also need an `ifup <iface>` here?
According to @champtar issue, this fixed the problem for him.

@champtar could you test if this works for you?

Reply:
Almost there, before this patch the interface is offline and traffic is not flowing:
![image](https://user-images.githubusercontent.com/3536755/116543635-171c9800-a8bc-11eb-85cc-e4c2e3fe1313.png)

With this patch the CLI status is almost correct and traffic is working as expected:
```
Interface status:
 interface wg_fr is online 00h:00m:00s, uptime 00h:01m:27s and tracking is not enabled
 interface wg_us is online 00h:00m:00s, uptime 00h:01m:27s and tracking is not enabled
```
But not the GUI one
![image](https://user-images.githubusercontent.com/3536755/116543576-08ce7c00-a8bc-11eb-98e9-ba4d57d21756.png)